libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.scss

```
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+watt-nav-list {
+  --watt-nav-list-color-secondary: #f8ad3c;
+
+  background-color: var(--watt-color-primary-dark);
+  display: block;
+
+  .mat-list-base {
+    padding-top: 0;
+  }
+
+  .mat-nav-list .mat-list-item {
+    border-left: var(--watt-space-xs) solid transparent;
+    color: var(--watt-color-neutral-white);
+    height: calc(10 * var(--watt-space-xs));
```

```suggestion
    height: auto;
```
And then add `
.mat-list-item-content {
     padding: var(--watt-space-s) var(--watt-space-m);
}
`